paddle/fluid/operators/reader/open_files_op.cc

```diff
@@ -47,28 +63,27 @@ class MultipleReader : public framework::ReaderBase {
   framework::Channel<size_t>* waiting_file_idx_;
   framework::Channel<size_t>* available_thread_idx_;
   framework::Channel<std::vector<framework::LoDTensor>>* buffer_;
-  mutable std::vector<framework::LoDTensor> local_buffer_;
+  mutable ThreadBufferMap thread_buffer_map_;
```

Comment:
I am not sure do we must have keyword `mutable` here?

Reply:
This has to be `mutable` because we have made `HasNext()` `const`. More practice is required to show whether it was a good decision to make it `const`. We may modify this part in further PRs.